config/cesm/machines/config_machines.xml

```diff
@@ -472,6 +472,7 @@ This allows using a different mpirun command to launch unit tests
       <arguments>
         <arg name="labelstdout">-p "%g:"</arg>
         <arg name="num_tasks"> -np {{ total_tasks }}</arg>
+        <arg name="zthreadplacement"> omplace -tm open64 -vv</arg>
```

Comment:
Needed when ESMF_AWARE_THREADING is off, not desirable when it is on. leaving it on for now.